templates/main.html

```diff
@@ -195,6 +204,7 @@ <h3>
     </div>
     <footer>
       <small data-i18n="footer-text">Google Code-in and the Google Code-in logo are trademarks of Google Inc.</small>
+      <small>The GitLab logo is licensed under <a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a> by GitLab Inc.</small>
```

Comment:
shouldnt this be i18n-able?

Reply:
Any idea how to use i18n with the link? If I include the link, it renders all the `<a href="...` as text on the page.

Maybe something like this would work, but it's not altogether pretty:

```
<small><span data-i18n="gitlab-copyright-1">The GitLab logo is licensed under </span><a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a><span data-i18n="gitlab-copyright-2"> by GitLab Inc.</span></small>

Reply:
^ That's ok.

Also add `<br>` after Google copyright.

Reply:
@blazeu done 👍